index.html

```diff
@@ -55,7 +55,7 @@ <h3>
                 <section data-background-image="images/TyPower.svg"
                          data-background-size="15%" data-background-position="right">
                     <h2>
-                        Teams presented today
+                        Teams presented today in more detail
```

Comment:
On my screen, this makes the heading split into two lines, pushing the "More information about all teams:" paragraph down out of the slide. How can we solve this?

Reply:
Should we remove this paragraph (as it's also present on the newly added slides)?

Reply:
I removed the "in more detail" part and adapt the space before the the overview link.